.github/workflows/build-test-package.yml

```diff
@@ -156,6 +156,12 @@ jobs:
         sudo rm -rf "$AGENT_TOOLSDIRECTORY"
         df -h
 
+    - name: 'Fetch build dependencies'
```

Comment:
By looking at log of the run, I realized that this was not needed - `ubuntu-22.04` already includes zstd and it is the latest version.

Reply:
I'm not sure I understand, the most recent run uses `ubuntu-20.04` and not `ubuntu-22.04`?

Reply:
I used 22.04 in ome-ngff module. In 20.04 apt says: `The following NEW packages will be installed: zstd`.

Reply:
Okay great, good catch 👍 let's stick with `ubuntu-20.04` for stability for now and plan to remove the zstd install step when we bump to `ubuntu-22.04` in the near future.